src/export/node_export_algorithm.rb

```diff
@@ -82,11 +87,98 @@ def run
 
   private
 
+  def offset_rotary_hinge_hubs(rotary_hinges, static_groups)
+    static_groups_as_sets_of_nodes = static_groups.map do |triangles|
+      nodes = Set.new
+      triangles.each do |triangle|
+        nodes << triangle.first_node
+        nodes << triangle.second_node
+        nodes << triangle.third_node
+      end
+      nodes
+    end
+
+    rotary_hinges.each do |node_pair|
+      static_groups_pair = static_groups_as_sets_of_nodes.select { |node_set| node_set.include?(node_pair[0]) && node_set.include?(node_pair[1]) }
```

Comment:
Also: Is `static_groups_pair` really a pair? It doesn't seem clear to me why we can assume that... if we can indeed maybe add a comment for explanation.

Reply:
I will add a runtime check. We can't assume it, but we will for now, since it keeps the insetting easier